[Bug]: radxa-dragon-q6a build failed

### What happened?

build image for radxa q6a failed with error:

```
gcc: error: unrecognized command-line option '-fmin-function-alignment=8'; did you mean '-flimit-function-alignment'?
make[4]: *** [/usr/src/linux-headers-6.18.2-current-qcs6490/scripts/Makefile.build:287: aic_load_fw/aic_bluetooth_main.o] Error 1
make[4]: *** Waiting for unfinished jobs....
  CC [M]  aic_load_fw/aic_compat_8800d80.o
gcc: error: unrecognized command-line option '-fmin-function-alignment=8'; did you mean '-flimit-function-alignment'?
make[4]: *** [/usr/src/linux-headers-6.18.2-current-qcs6490/scripts/Makefile.build:287: aic_load_fw/aicbluetooth.o] Error 1
gcc: error: unrecognized command-line option '-fmin-function-alignment=8'; did you mean '-flimit-function-alignment'?
make[4]: *** [/usr/src/linux-headers-6.18.2-current-qcs6490/scripts/Makefile.build:287: aic8800_fdrv/rwnx_wakelock.o] Error 1
make[4]: *** Waiting for unfinished jobs....
  CC [M]  aic8800_fdrv/rwnx_cmds.o
gcc: error: unrecognized command-line option '-fmin-function-alignment=8'; did you mean '-flimit-function-alignment'?
gcc: error: unrecognized command-line option '-fmin-function-alignment=8'; did you mean '-flimit-function-alignment'?
make[4]: *** [/usr/src/linux-headers-6.18.2-current-qcs6490/scripts/Makefile.build:287: aic_load_fw/aicwf_usb.o] Error 1
make[4]: *** [/usr/src/linux-headers-6.18.2-current-qcs6490/scripts/Makefile.build:287: aic8800_fdrv/rwnx_msg_tx.o] Error 1
gcc: error: unrecognized command-line option '-fmin-function-alignment=8'; did you mean '-flimit-function-alignment'?
make[4]: *** [/usr/src/linux-headers-6.18.2-current-qcs6490/scripts/Makefile.build:287: aic_load_fw/aic_txrxif.o] Error 1
gcc: error: unrecognized command-line option '-fmin-function-alignment=8'; did you mean '-flimit-function-alignment'?
make[4]: *** [/usr/src/linux-headers-6.18.2-current-qcs6490/scripts/Makefile.build:287: aic8800_fdrv/rwnx_msg_rx.o] Error 1
gcc: error: unrecognized command-line option '-fmin-function-alignment=8'; did you mean '-flimit-function-alignment'?
make[4]: *** [/usr/src/linux-headers-6.18.2-current-qcs6490/scripts/Makefile.build:287: aic_load_fw/aicbluetooth_cmds.o] Error 1
gcc: error: unrecognized command-line option '-fmin-function-alignment=8'; did you mean '-flimit-function-alignment'?
make[4]: *** [/usr/src/linux-headers-6.18.2-current-qcs6490/scripts/Makefile.build:287: aic8800_fdrv/rwnx_utils.o] Error 1
gcc: error: unrecognized command-line option '-fmin-function-alignment=8'; did you mean '-flimit-function-alignment'?
make[4]: *** [/usr/src/linux-headers-6.18.2-current-qcs6490/scripts/Makefile.build:287: aic_load_fw/aic_compat_8800d80.o] Error 1
make[3]: *** [/usr/src/linux-headers-6.18.2-current-qcs6490/scripts/Makefile.build:556: aic_load_fw] Error 2
make[3]: *** Waiting for unfinished jobs....
gcc: error: unrecognized command-line option '-fmin-function-alignment=8'; did you mean '-flimit-function-alignment'?
make[4]: *** [/usr/src/linux-headers-6.18.2-current-qcs6490/scripts/Makefile.build:287: aic8800_fdrv/rwnx_cmds.o] Error 1
make[3]: *** [/usr/src/linux-headers-6.18.2-current-qcs6490/scripts/Makefile.build:556: aic8800_fdrv] Error 2
make[2]: *** [/usr/src/linux-headers-6.18.2-current-qcs6490/Makefile:2010: .] Error 2
make[1]: *** [/usr/src/linux-headers-6.18.2-current-qcs6490/Makefile:248: __sub-make] Error 2
make[1]: Leaving directory '/var/lib/dkms/aic8800-usb/4.0+git20250410.b99ca8b6-5/build/USB/driver_fw/drivers/aic8800'
make: *** [Makefile:248: __sub-make] Error 2
make: Leaving directory '/usr/src/linux-headers-6.18.2-current-qcs6490'
[💥] Error 1 occurred in main shell [ at /data/build/src/armbian-build/lib/functions/logging/runners.sh:15
   chroot_sdcard_apt_get_install() --> lib/functions/logging/runners.sh:15
   post_install_kernel_debs__install_aic8800_dkms_package() --> extensions/radxa-aic8800.sh:52
       post_install_kernel_debs() --> <extension_magic>:328
          call_extension_method() --> lib/functions/general/extensions.sh:67
   install_distribution_agnostic() --> lib/functions/rootfs/distro-agnostic.sh:392
                do_with_logging() --> lib/functions/logging/section-logging.sh:81
         build_rootfs_and_image() --> lib/functions/main/rootfs-image.sh:31
   full_build_packages_rootfs_and_image() --> lib/functions/main/default-build.sh:36
          do_with_default_build() --> lib/functions/main/default-build.sh:42
         cli_standard_build_run() --> lib/functions/cli/cli-build.sh:25
        armbian_cli_run_command() --> lib/functions/cli/utils-cli.sh:136
                 cli_entrypoint() --> lib/functions/cli/entrypoint.sh:208
                           main() --> ./compile.sh:50
 ]
```

### How to reproduce?

tag: v26.2.0-trunk.552
build command :
```
./compile.sh BOARD=radxa-dragon-q6a \
BRANCH=current BUILD_DESKTOP=no BUILD_MINIMAL=no \
KERNEL_CONFIGURE=no RELEASE=noble USE_CCACHE=no \
INSTALL_HEADERS=yes EXTERNAL_NEW=compile EXTERNAL=yes  \
ARTIFACT_IGNORE_CACHE=yes\
DISABLE_IPV6=false NAMESERVER=192.168.27.1 \
DOWNLOAD_MIRROR=china 
```


### Branch

other (only relevant in corner cases)

### On which host OS are you running the build script and observing this problem?

Ubuntu 24.04 Noble

### Are you building on Windows WSL2?

- [ ] Yes, my Ubuntu/Debian/OtherOS is running on WSL2

### Relevant log URL

https://paste.armbian.com/vinezugaka

### Code of Conduct

- [x] I agree to follow this project's Code of Conduct

--- a/extensions/radxa-aic8800.sh
+++ b/extensions/radxa-aic8800.sh
@@ -48,6 +48,11 @@ function post_install_kernel_debs__install_aic8800_dkms_package() {
 	esac
 	use_clean_environment="yes" chroot_sdcard "wget ${aic8800_firmware_url} -P /tmp"
 	display_alert "Install aic8800 packages, will build kernel module in chroot" "${EXTENSION}" "info"
+	if [[ "${RELEASE}" == "noble" ]]; then
+		display_alert "Installing gcc-14 for DKMS build" "${EXTENSION}" "info"
+		use_clean_environment="yes" chroot_sdcard_apt_get_install "gcc-14"
+		use_clean_environment="yes" chroot_sdcard "update-alternatives --install /usr/bin/gcc gcc /usr/bin/gcc-14 100"
+	fi
 	declare -ag if_error_find_files_sdcard=("/var/lib/dkms/aic8800*/*/build/*.log")
 	use_clean_environment="yes" chroot_sdcard_apt_get_install "/tmp/${aic8800_dkms_file_name} /tmp/aic8800-firmware_${latest_version}_all.deb"
 	use_clean_environment="yes" chroot_sdcard "rm -f /tmp/aic8800*.deb"

